content/docs/studio/troubleshooting.md

```diff
@@ -206,11 +203,11 @@ There are two possible reasons for this:
    columns that you select in the
    [**Columns** setting](/doc/studio/user-guide/projects-and-experiments/configure-a-project#columns).
 
-   **What if there are more than 200 columns?** Currently Iterative Studio
-   cannot import over 200 columns. If you have a large repository (with more
-   than 200 columns), one solution is to split the
+   **What if the repository has more than 200 columns?** Currently Iterative
```

Comment:
@amritghimire What is the current number for this limit (instead of 200)?

Reply:
It is by default 5k. But, probably changed by @iterative/platform

Reply:
It relies on PARSING_LIMITS

Reply:
We use default `PARSING_LIMITS` value in production, so 5k

Reply:
Thanks @amritghimire @jesper7

Reply:
@amritghimire @jesper7 I can see that the PARSING_LIMITS was changed to 2k in the itops repo, and that change was later reverted. In the Studio repo, the PARSING_LIMITS has different values for soft/hard/all_columns. Can you pls point to which configuration gets finally applied?

Reply:
@mvshmakov Since you are working on https://github.com/iterative/studio/issues/6058, can you pls confirm the numbers here?

Reply:
The default parsing limits in `docker-compose/base.yaml` is `{"commits":{"soft":3000,"hard":5000},"columns":{"soft":100,"hard":200},"all_columns":3000}`.

The default value that backend fallbacks to if nothing was specified at all is `{"commits": {"soft": 3000, "hard": 5000},"columns": {"soft": 200, "hard": 200},"all_columns": 5000}`.

> In the Studio repo, the PARSING_LIMITS has different values for soft/hard/all_columns. Can you pls point to which configuration gets finally applied?

From the docs:

```txt
Soft limits: max amount of commits / columns to import during the first parse.
Hard limits: max amount of commits / columns to import on subsequent parses.
All columns: max amount of entries in files/params/metrics to track for display
```

So, depending on whether we apply `base.yaml` env variables, it is either 3000 or 5000. @iterative/platform do we propagate `docker-compose/base.yaml` env variables by default when deploying to the production?

Reply:
No, `docker-compose` configuration is separate from prod. To propagate someone has to do this in `config.env` files 
- [prod](https://github.com/iterative/itops/blob/main/k8s/prod/studio/config.env)
- [prod-uk](https://github.com/iterative/itops/blob/main/k8s/uk/studio/default/config.env)
- [dev](https://github.com/iterative/itops/blob/main/k8s/dev/studio/config.env)